src/ontology/mondo-ingest.Makefile

```diff
@@ -163,11 +163,6 @@ $(COMPONENTSDIR)/icd10who.owl: $(TMPDIR)/icd10who_relevant_signature.txt | compo
 		remove -T config/properties.txt --select complement --select properties --trim true \
 		annotate --ontology-iri $(URIBASE)/mondo/sources/icd10who.owl --version-iri $(URIBASE)/mondo/sources/$(TODAY)/icd10who.owl -o $@; fi
 
-.PHONY: component-download-gard.owl
-component-download-gard.owl: | $(TMPDIR)
```

Comment:
## The removal
I checked and it is redundant, and the goal is 100% the same in the `Makefile`.